lib/html/static/js/search.js

```diff
@@ -77,28 +77,36 @@ var FileLoader = ( function () {
     };
 
 
-    var selectLine = function (num) {
+    this.selectLine = function (num) {
         /* Highlights and scrolls to the line of code with the provided
         * number. */
-        // insert highlight tags to selected line
-        var lines = $('#filecode').html().split('\n');
-        if (num < lines.length) {
-            lines[num - 1] = mark[0] + lines[num - 1] + mark[1];
+        num = parseInt(num);
+
+        if (previous_search) {
+            loadFile("", num)
```

Comment:
Note that https://github.com/oliver-sanders/rose/pull/5/files#diff-cb9830bbb3b655703396b58b67a92e6b adds javascript syntax checking to the test battery for `sphinx/_static` this could be expanded to `lib/html/static/js` as well.